packages/cli/src/commands/test.ts

```diff
@@ -56,7 +62,7 @@ export default class Test extends AuthCommand {
       description: 'list all checks but don\'t run them.',
     }),
     timeout: Flags.integer({
-      default: 300,
+      default: DEFAULT_CHECK_RUN_TIMEOUT_SECONDS,
```

Comment:
@clample , if Checkly should always report a result within 240s, is it possible to limit users to configure a value lower than `240`?

Reply:
I think that we should continue letting users set any timeout that they'd like. It may make sense for their setup to set a shorter timeout - for example, if they only have API checks which don't take so long.

I think that the default of 300s makes sense, though. This is long enough to make sure that the check runs should be completed, plus a buffer for any latency in sending over the check result. This timeout should only ever be reached if checkly is experiencing issues.